docs/reference/ilm/overview-ilm.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[testenv="basic"]
+[[overview-index-lifecycle-management]]
+== Index lifecycle management overview
+
+++++
+<titleabbrev>Overview</titleabbrev>
+++++
+
+You can create and apply {ilm-cap} ({ilm-init}) policies to automatically manage your indices 
+according to your performance, resiliency, and retention requirements. 
+Index lifecycle policies can trigger actions such as rollover, shrink, freeze, and delete. 
```

Comment:
I'm not sure a new user would know what these actions are or what they mean.
It may be more helpful to do one of the following:

* Remove this sentence
* Add links to the example actions
* Reword this to go into a bit more depth